spec/models/piece_spec.rb

```diff
@@ -1,4 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe "capture!" do
+    it "return false if specified coordinates occupied by friendly piece" do
+      white_pawn = FactoryGirl.create(:pawn, :white, x_position: 3, y_position: 0, captured: false)
+      other_white_pawn = FactoryGirl.create(:pawn, :white, x_position: 4, y_position: 1, captured: false)
+
+      expect(other_white_pawn.capture!(white_pawn.x_position,white_pawn.y_position)).to eq false
+    end
+
+    it "return true if specified coordinates occupied by opponent's piece" do
+      black_rook = FactoryGirl.create(:rook, :black, x_position: 4, y_position: 0, captured: false)
+      white_pawn = FactoryGirl.create(:pawn, :white, x_position: 4, y_position: 1, captured: false)
+
+      expect(white_pawn.capture!(black_rook.x_position,black_rook.y_position)).to eq true
```

Comment:
This test is not adequately testing whether your capture logic is working.  It's just testing whether or not you explicitly returned true.  There is no actual capture logic implemented in your code.